assertion/function/assertiontree/backprop_util.go

```diff
@@ -267,14 +267,6 @@ func handleErrorReturns(rootNode *RootAssertionNode, retStmt *ast.ReturnStmt, re
 	errRetExpr := results[errRetIndex]     // n-th expression
 	nonErrRetExpr := results[:errRetIndex] // n-1 expressions
 
-	// check if the error return is at all guarding any nilable returns, such as pointers, maps, and slices
-	for _, r := range nonErrRetExpr {
-		if util.ExprBarsNilness(rootNode.Pass(), r) {
-			// no need to further analyze and create triggers
-			return
```

Comment:
Safe to remove this logic, since `ExprBarsNilness` is checked before creating the consume triggers anyway.